### PR TITLE
Enable registration button for barcamp site

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by kelnera** [comment]
@ramereth Never mind! @jldugger helped me figure it out. I'll fix #28 and a couple other things and then it should be ready to go.

File: content/pages/about.rst [hunk 1]
```diff
@@ -76,20 +76,21 @@ Directions
...
-Parking may be a little bit tricky this year, as the parking lot 
+Parking may be a little bit tricky this year, as the parking lot
 just north of Kelley is closed, and there is a footbal scrimmage
-on the same day as Barcamp.  We recommend you park in the 
+on the same day as Barcamp.  We recommend you park in the
 highlighted parking lots close to Kelley on the map below! Parking
 is free everywhere, so if you find a space go ahead and take it.
 
-.. figure:: ../theme/img/parking2.png
+.. image:: /images/BBCParkingMap.png
```
Comment (ramereth) [inline content/pages/about.rst]: This should stay figure I think.

Comment (ramereth) [inline content/pages/about.rst]: Actually, the image path is wrong too :(

**Comment by kelnera** [comment]
@ramereth @Kennric @osuosl/web-reviewers I believe I've made all the changes we need to update the website. They're mostly little fixes but if someone could review them, that would be great!

File: content/pages/about.rst (hunk 1 above)
Comment (kelnera) [inline content/pages/about.rst]: I think we've resolved the issue. It took a little digging but it's working now on the Jenkins site. If you see any issues though let me know.

**Pull request merged**